repositories/user_repository.js

```diff
@@ -1,35 +1,40 @@
 const { getKnex } = require("../db");
- 
-async function getAllUsers() {
-	const knex = getKnex();
+const knex = getKnex(); async function getAllUsers() {
+
 	return await knex("users").select("id", "name", "languages");
 }
 async function createNewUser(name, email, about, languages) {
-	const knex = getKnex();
-	const [user] = await knex("users").insert({ name, email, about, languages }).returning("id");
-	return user.id;
+	const [user] = await knex("users").insert({ name, email, about, languages }).returning("*");
+	return user;
 }
 async function getUserInfoById(id) {
-	const knex = getKnex();
 	return await knex("users").where("id", id).first();
 }
 async function getUserRolesById(id) {
-	const knex = getKnex();
 	return await knex("roles")
 		.join("user_roles", "roles.id", "user_roles.role_id")
 		.where("user_roles.user_id", id)
 		.select("roles.role_name", "roles.id");
 }
 async function updateUserById(id, name, about, languages) {
-	const knex = getKnex();
 	return await knex("users")
 		.where("id", id)
 		.update({name, about, languages })
 		.returning("*");
 	
 }
 async function deleteUserById(id) {
-	const knex = getKnex();
 	return await knex("users").where("id", id).del();
 }
-module.exports = {  getAllUsers, createNewUser, getUserInfoById, getUserRolesById, updateUserById, deleteUserById };
\ No newline at end of file
+async function getUserByEmail(email) {
+	return await knex("users")
+		.where({ email })
+		.first();
+}
+async function updateUserFirebaseId(id, firebase_id) {
```

Comment:
is it possible to make a reusable method? 
like updateUserById(... some args that we want to update)